tests/test_config.yaml

```diff
@@ -5,18 +5,18 @@ tests:
     ct_json_elements: 13
     ct_data_frame_rows: 141
     ct_graph_queries_owl:
-      query_count_equivalent_classes: 90
+      query_count_equivalent_classes: 84
```

Comment:
Why did this happen?

Reply:
All of these changes were made in the previous PR (Validation) since many of the test files changed [as seen here](https://github.com/mapping-commons/sssom-py/pull/283/files#diff-a1794df9a43ac98ef5982e745aaf02c8cc7d2ff2d91878e34ee621d12b13d439). But it seemed like that never travelled to the main branch. When I pulled from main, those changes never made it to this branch. I'm guessing since I had these branches started in parallel, something got missed somewhere.